Source/Blazorise/Select.razor.cs

```diff
@@ -209,6 +211,25 @@ public bool Multiple
         /// </summary>
         [Parameter] public Expression<Func<IReadOnlyList<TValue>>> SelectedValuesExpression { get; set; }
 
+        /// <summary>
+        /// Gets or sets loading property.
+        /// </summary>
+        [Parameter]
+        public bool Loading
+        {
+            get => loading;
+            set
+            {
+                loading = value;
+                Disabled = value;
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets loading property.
+        /// </summary>
+        [Parameter] public bool AllowClear { get; set; }
```

Comment:
I'm not sure about this property. It's only used on AntDesign and nowhere else. It's best to either remove it or implement the same behavior for all providers.